backend/src/rtp.rs

```diff
@@ -1775,29 +1837,34 @@ impl Endpoint {
     ) -> Option<Packet<Vec<u8>>> {
         let tcc_sender = &mut self.tcc_sender;
         let rtx_sender = &mut self.rtx_sender;
-        let rtx = rtx_sender.resend_as_rtx(ssrc, seqnum, || tcc_sender.increment_seqnum())?;
-        self.encrypt_and_send_rtp(rtx, now)
+        let rtx = rtx_sender
+            .resend_as_rtx(ssrc, seqnum, || tcc_sender.increment_seqnum())
+            .ok_or_else(|| EndpointError::UndeterminedError("Packet Error".to_string()))
```

Comment:
I kind of see this as a `ResendAsRtxError` generic error. Not undetermined.

Reply:
Should I create a another error variant for handling the `RTXError`

Reply:
No